Documentation/Contributors/CLAs/corporate-cla-agi-v1.0.txt

```diff
@@ -0,0 +1,147 @@
+                      Analytical Graphics, Inc.
+  Software Grant and Corporate Contributor License Agreement ("Agreement")
+           http://www.agi.com/licenses/corporate-cla-agi-v1.0.txt
```

Comment:
Yeah, I saw that too.  I wasn't sure if it'd be acceptable to use a URL shortener.